esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2981,9 +2981,9 @@ bool CFileSprayEx::onDropZoneFileSearch(IEspContext &context, IEspDropZoneFileSe
         if (!directory.length())
             throw MakeStringException(ECLWATCH_INVALID_INPUT, "DropZone Directory not found for %s.", dropZoneName);
 
-        IpAddress ipToMatch;
-        ipToMatch.ipset(dropZoneServerReq);
-        if (ipToMatch.isNull())
+        IpAddress ipAddress;
+        ipAddress.ipset(dropZoneServerReq);
+        if (ipAddress.isNull())
```

Comment:
This check should also probably go, but that should wait for a later PR since searching dropzones that are implemented as blob storage needs more work before it will work.